src/libcore/lib.rs

```diff
@@ -103,9 +101,6 @@
 #![feature(untagged_unions)]
 #![feature(unwind_attributes)]
 
-#![cfg_attr(stage0, allow(unused_attributes))]
-#![cfg_attr(stage0, feature(never_type))]
```

Comment:
Given https://github.com/rust-lang/rust/issues/49593, do we still want this?

Reply:
Perhaps not yeah, but we can always add this back if the never type gets destabilized